src/codeu/chat/client/commandline/Chat.java

```diff
@@ -370,6 +404,249 @@ private ConversationContext find(String title) {
       }
     });
 
+    // C-INTEREST-LIST (list conversations)
+    //
+    // Add a command that will print all conversations the user is interested in when the user enters
+    // "c-interest-list" while on the user panel.
+    //
+    panel.register("c-interest-list", new Panel.Command() {
+      @Override
+      public void invoke(List<String> args) {
+        Set<Uuid> interestedConvos = convoInterestMap.get(user.user.id);
+        if(interestedConvos != null){
+          for (final Uuid convoID : interestedConvos) {
+            final ConversationContext conversation = findConversation(convoID);
```

Comment:
Ah, I meant in `findConversations`. You can make `findConversations` O(1) to speed this up.